jerry-core/vm/vm-stack.h

```diff
@@ -28,13 +28,13 @@
 /**
  * Create context on the vm stack.
  */
-#define VM_CREATE_CONTEXT(type, end_offset) ((ecma_value_t) ((type) | ((end_offset) << 6)))
+#define VM_CREATE_CONTEXT(type, end_offset) ((ecma_value_t) ((type) | ((end_offset) << 8)))
```

Comment:
Seems 7 is enough. I would not increase it to 8, because it might be misleading.